src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -95,21 +94,17 @@ public void RunTests(IEnumerable<string> sources, IRunContext runContext, IFrame
                     if (!Path.IsPathRooted(assemblyName))
                         assemblyName = Path.Combine(Environment.CurrentDirectory, assemblyName);
 
-                    TestLog.Info("Running all tests in " + assemblyName);
-
                     RunAssembly(assemblyName, TestFilter.Empty);
                 }
                 catch (Exception ex)
                 {
-                    if (ex is TargetInvocationException)
-                        ex = ex.InnerException;
+                    if (ex is TargetInvocationException) { ex = ex.InnerException; }
```

Comment:
Please keep to our layout rather than making changes purely for your own style preferences.